src/components/Thumbnail/Thumbnail.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import {thumbnailDriverFactory} from './Thumbnail.driver';
+import {Thumbnail} from './';
+import {createDriverFactory} from 'wix-ui-test-utils/driver-factory';
+import {isEnzymeTestkitExists} from 'wix-ui-test-utils/enzyme';
+import {isTestkitExists} from 'wix-ui-test-utils/vanilla';
+import {mount} from 'enzyme';
+import Check from 'wix-ui-icons-common/Check';
+import {thumbnailTestkitFactory} from '../../testkit';
+import {thumbnailTestkitFactory as enzymeThumbnailTestkitFactory} from '../../testkit/enzyme';
+
+describe('Thumbnail', () => {
+  const createDriver = createDriverFactory(thumbnailDriverFactory);
+
+  describe('title prop', () => {
+    it('should render the given title', () => {
+      const driver = createDriver(<Thumbnail title="hello"/>);
+      expect(driver.titleDriver().getText()).toEqual('hello');
+    });
+
+    it('should have bold text', () => {
+      const driver = createDriver(<Thumbnail title="hello"/>);
+      expect(driver.titleDriver().isBold()).toEqual(true);
+    });
+  });
+
+  describe('description prop', () => {
+    it('should not have description by default', () => {
+      const driver = createDriver(<Thumbnail title="hello"/>);
+      expect(driver.hasDescription()).toBe(false);
+    });
+
+    it('should render the description', () => {
+      const driver = createDriver(<Thumbnail title="hello" description="hey"/>);
+      expect(driver.getDescription()).toEqual('hey');
```

Comment:
add check that `hasDescription()` returns true

Reply:
Feels redundant.
The only reasonable explanation will be to test the `hasDescription` method.
Will add it.